crawler/crawler.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/libp2p/go-libp2p/core/protocol"
 
 	logging "github.com/ipfs/go-log"
+	//lint:ignore SA1019 TODO migrate away from gogo pb
```

Comment:
:nose:

Reply:
Just do it!

Reply:
I spent 10 minutes and failed.

Reply:
What’s the problem? I did this for libp2p and it should be quite straightforward. Maybe I can help?

Reply:
There's a makefile that doesn't work. It needs you to fetch certain dependencies before running `make`. That script needs to get updated (or removed in favor of gogenerate). Bonus points for having CI check the output.